packages/gasket-plugin-elastic-apm/lib/create.js

```diff
@@ -0,0 +1,24 @@
+const { name, version, devDependencies } = require('../package.json');
+
+/** @type {import('@gasket/core').HookHandler<'create'>} */
+module.exports = function create(gasket, { pkg, files, gasketConfig }) {
+  const generatorDir = `${__dirname}/../generator`;
+
+  gasketConfig.addPlugin('pluginElasticApm', name);
+
+  pkg.add('dependencies', {
+    [name]: `^${version}`,
+    'dotenv': devDependencies.dotenv,
+    'elastic-apm-node': devDependencies['elastic-apm-node']
+  });
+
+  pkg.extend((current) => {
+    return {
+      scripts: {
+        start: `node --import ./setup.js & ${current.scripts.start}`
```

Comment:
`&` is going to be a different process..

I think what you want here is to set [NODE_OPTIONS](https://nodejs.org/api/cli.html#-r---require-module) per examples in the [APM docs](https://www.elastic.co/guide/en/apm/agent/nodejs/current/nextjs.html#_step_2_start_the_apm_agent_2):
```suggestion
        start: `NODE_OPTIONS=--import=./setup.js ${current.scripts.start}`
```

Can you give that a shot? It should work with either Next or API start scripts